controllers/users.js

```diff
@@ -0,0 +1,13 @@
+const { findUsers } = require("../models/users");
+
+function getUsers(req,res,next) {
+  findUsers()
```

Comment:
I'm not the biggest fan of this as a variable name if I'm being picky. One convention I like to follow is naming models by the type of request eg. if the model is a GET request it's getUsers, then the controller is fetching from the model so would be fetchUsers.

Reply:
I was just using the naming convention they've been using in the lectures. I will rethink 🧐